### PR TITLE
Skip Decoders when sending join-req to integration

### PR DESCRIPTION
**Pull request by michaeldjeffrey**
Join frames don't have payloads, so there's nothing to decode.

### CODE REVIEW

**Review by dpezely — DISMISSED** [review]
We have a lot of repetition within test code, so at some point we should consider generic/prepopulated maps for `wait_for_console_*()` and insert just the override values.

File: src/channels/router_channel.erl [hunk 1]
```diff
@@ -307,7 +296,13 @@ encode_data(Decoder, #{payload := Payload, port := Port} = TemplateArgs, Channel
                     payload => base64:encode(Payload)
                 })
             )
-    end.
+    end;
+encode_data(_Decoder, TemplateArgs, Channel) ->
+    lager:info("missing payload and port [device: ~p]", [?MODULE:device_id(Channel)]),
+    router_channel_utils:maybe_apply_template(
```
Comment (macpie) [inline src/channels/router_channel.erl:302]: I wonder if we should even apply template?

**Review by mikev — APPROVED** [review]

File: src/channels/router_channel.erl (hunk 1 above)
Comment (michaeldjeffrey) [reply src/channels/router_channel.erl:302]: That's a fair point. They still might be trying to grab `payload` or the likes and will fail on that.

File: src/channels/router_channel.erl [hunk 2]
```diff
@@ -307,7 +296,13 @@ encode_data(Decoder, #{payload := Payload, port := Port} = TemplateArgs, Channel
                     payload => base64:encode(Payload)
                 })
             )
-    end.
+    end;
+encode_data(_Decoder, TemplateArgs, _Channel) ->
+    lager:info("encoding join [device: ~p] [channel: ~p]", [
```
Comment (macpie) [inline src/channels/router_channel.erl:301]: Lets just take that log line down to debug
```suggestion
    lager:debug("encoding join [device: ~p] [channel: ~p]", [
```

**Review by dpezely — APPROVED** [review]

**Review by macpie — APPROVED** [review]

**Pull request merged**